resources/assets/components/CampaignSingle/index.js

```diff
@@ -24,15 +24,37 @@ class CampaignSingle extends React.Component {
     this.deletePost = this.deletePost.bind(this);
     this.filterPosts = this.filterPosts.bind(this);
     this.getPostsByPaginatedLink = this.getPostsByPaginatedLink.bind(this);
+    this.createTagSlug = this.createTagSlug.bind(this);
+    this.getPostsByTag = this.getPostsByTag.bind(this);
   }
 
   componentDidMount() {
     this.getPostsByStatus('accepted', this.props.campaign.id);
   }
 
-  // Filter posts based on status.
-  filterPosts(status) {
-    this.getPostsByStatus(status.toLowerCase(), this.props.campaign.id);
+  // Filter posts based on status or tag(s).
+  filterPosts(filter) {
+    // If the filter is a status, make API call to get posts by status.
+    if (filter === 'Pending' || filter === 'Accepted' || filter === 'Rejected') {
```

Comment:
You can use [`include()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) here to see if the filter is in an array of values, instead of multiple or statements

Reply:
oh awesome! So much cleaner.